src/Akka.Persistence.MongoDb.Tests/MongoDbAllEventsSpec.cs

```diff
@@ -19,6 +19,8 @@ public class MongoDbAllEventsSpec: AllEventsSpec, IClassFixture<DatabaseFixture>
     {
         private static Config CreateSpecConfig(DatabaseFixture databaseFixture, int id)
         {
+            var s = databaseFixture.ConnectionString.Split('?');
```

Comment:
Is there a way we could standardize this into a function, so we don't have to do this string manipulation the same way in ~12 different tests?